snare/server.py

```diff
@@ -0,0 +1,106 @@
+import asyncio
```

Comment:
unused imports

Reply:
I think we need asyncio for all `async def` syntax, what do you think